src/modules/home/constants/category-icon-map.constant.ts

```diff
@@ -0,0 +1,57 @@
+import { PiTent } from 'react-icons/pi';
```

Comment:
For just two icons, use local SVGs instead of using library, it minimizes your bundle size and ensures better performance. Maybe find this kind of icon from mui.

Reply:
I tried many other icons but they did not look consistent together with mui icons. Mui also does not have tent icons so I could not use it.